### PR TITLE
Display groups on project overview page

### LINKED ISSUE #4730: Show groups list on project overview
**What would you like to be added? / User Story**:

New [Groups](https://github.com/kubermatic/dashboard/pull/4712) feature has been added therefore we need to show groups similar to like `Cluster and External Cluster` tab are shown.

https://user-images.githubusercontent.com/17727069/181701346-f8ed146f-d173-41ed-abf1-ad1e977410bb.mp4

### PR DESCRIPTION
**Pull request by KhizerRehan**
Signed-off-by: khizerrehan <khizerrehan92@gmail.com>

**What this PR does / why we need it**:

This PR displays list of groups on project overview page in enterprise edition + has permission of `owners`. 

**Enterprise Edition**

**Role: owner**

owners

**Role: editor/viewer**

permission-otther-than-owner

**Note**: In Case multiple groups UI separates groups by. ","  and it automatically wraps to new row when reaches to end of line:

multiple-groups

**Community Edition**

**Role: owner**

communtiy-editon

**Role: editor/viewer**

community-edition-other-than-owners

**Which issue(s) this PR fixes**:

Fixes #4730 

**What type of PR is this?**
/kind feature
/kind design

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
 Displays list of groups on project overview page in enterprise edition + has permission of owners
```

**Documentation**:

```documentation
TBD
```

### CODE REVIEW

File: src/app/project-overview/members-overview/template.html [hunk 1]
```diff
@@ -20,9 +20,7 @@
       <div fxLayout="row"
            fxLayoutAlign=" center"
            fxLayoutGap="10px">
-        <span>{{showMembers ? 'Owners & Members' : 'Owners'}}</span>
-        <a class="members-link"
-           [routerLink]="['../members']"><i class="km-icon-next km-icon-mask km-no-margin"></i></a>
+        <span>{{showMembersAndGroups ? 'Owners, Members & Groups' : 'Access'}}</span>
```
Comment (Waseem826) [inline src/app/project-overview/members-overview/template.html:23]:
```suggestion
        <span>{{showMembersAndGroups ? 'Owners, Members & Groups' : 'Owners'}}</span>
```

File: src/app/project-overview/members-overview/template.html [hunk 2]
```diff
@@ -33,12 +31,37 @@
...
+        <i class="km-icon-mask km-icon-group"></i>
+        <div>{{groupNames}}</div>
+      </div>
+    </ng-container>
   </mat-card-content>
+
+  <div *ngIf="showMembersAndGroups"
+       fxLayout="row"
+       fxLayoutAlign=" center"
+       fxLayoutGap="15px">
```
Comment (Waseem826) [inline src/app/project-overview/members-overview/template.html:55]: There is only child in this `div` so why do we need `fxLayoutGap`?

File: src/app/project-overview/members-overview/component.ts [hunk 3]
```diff
@@ -24,7 +27,11 @@ import {Member} from '@shared/entity/member';
 export class MembersOverviewComponent {
   @Input() project: Project;
   @Input() members: Member[] = [];
-  @Input() showMembers = true;
+  @Input() groups: Group[] = [];
+  @Input('showMembers') showMembersAndGroups = true;
```
Comment (Waseem826) [inline src/app/project-overview/members-overview/component.ts:31]: I think we should not use alias here.

**Review by Waseem826 — CHANGES_REQUESTED** [review]
Some feedback. PTAL

File: src/app/project-overview/members-overview/template.html (hunk 1 above)
Comment (KhizerRehan) [reply src/app/project-overview/members-overview/template.html]: It's been decided to change the headline to "**Access**" for everyone, the icons can get tooltips.

**Review by Waseem826 — APPROVED** [review]

**Pull request merged**